Fibonachi.java

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
This pull-request must not contain changes in any files which don't relate to the current task (sequence)